src/components/CardGrid/CardGrid.tsx

```diff
@@ -49,51 +49,48 @@ function CardGrid({
   accessModel,
   isLoggedIn,
   hasSubscription,
+  hasLoadMore,
+  loadMore,
+  onCardClick,
+  onCardHover,
 }: CardGridProps) {
   const breakpoint: Breakpoint = useBreakpoint();
   const posterAspect = parseAspectRatio(playlist.shelfImageAspectRatio);
   const visibleTiles = cols[breakpoint] + parseTilesDelta(posterAspect);
   const [rowCount, setRowCount] = useState(INITIAL_ROW_COUNT);
 
+  const defaultLoadMore = () => setRowCount((current) => current + LOAD_ROWS_COUNT);
```

Comment:
What was the reason for moving these to consts?

Reply:
The reason was to modify the behaviour of the infinite scroll. For series we need to load additional data when scrolling and for simple recommendations we just show extra items when scrolling (but we don't need to load them).